pymc/smc/smc.py

```diff
@@ -192,7 +192,7 @@ def initialize_population(self) -> Dict[str, np.ndarray]:
         return cast(Dict[str, np.ndarray], result)
 
     def _initialize_kernel(self):
-        """Create variables and logp function necessary to run kernel
+        """Create variables and logp function necessary to run SMC_kernel
```

Comment:
```suggestion
        """Create variables and logp function necessary to run SMC kernel
```

The alias with the underscore that gets rendered as a link only works if it is in the type section of a parameter description:

```
Parameters
----------
parameter_name : **parameter type**
    parameter description
```

Thus, it won't work here and we should use normal text.

Reply:
Thank you! Very helpful - I was reverse engineering a bit based on other documentation. Is it true for all aliases?

Reply:
Yes, true for all aliases defined in `conf.py`